python/lsst/analysis/ap/nb_utils.py

```diff
@@ -19,11 +19,18 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-__all__ = ["make_simbad_link"]
+__all__ = ["make_simbad_link", "compare_sources"]
 
 import astropy.coordinates as coord
+from astropy.coordinates import SkyCoord, match_coordinates_sky
 from astroquery.simbad import Simbad
 import astropy.units as u
+import os
+import pandas as pd
+import numpy as np
+
+from lsst.analysis.ap import plotImageSubtractionCutouts as pisc
```

Comment:
Please don't abbreviate this at all. Just write out the full name: you only need it a handful of times, it's not worth the confusion of an unknown abbreviation.